src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2005,10 +2025,12 @@ public partial struct RuntimeMethodHandle : System.Runtime.Serialization.ISerial
     }
     public partial struct RuntimeTypeHandle : System.Runtime.Serialization.ISerializable
     {
+        private object _dummy;
         public System.IntPtr Value { get { throw null; } }
         public override bool Equals(object obj) { throw null; }
         public bool Equals(System.RuntimeTypeHandle handle) { throw null; }
         public override int GetHashCode() { throw null; }
+        [System.CLSCompliantAttribute(false)]
```

Comment:
You can delete this unnecessary CLSCompliantAttribute attribute. I have deleted it in the implementation already.